utils/web_scraper.py

```
@@ -18,9 +18,96 @@
+                    logger.info(f"✅ 已强制补全缺失配置文件: {file_name}")
+                    fixed_count += 1
+                except Exception as e:
+                    logger.warning(f"❌ 写入配置文件 {file_name} 失败: {e}")
+            else:
+                # 检查文件是否为空或损坏 (可选)
+                try:
+                    if file_path.stat().st_size == 0:
+                        with open(file_path, "w", encoding="utf-8") as f:
+                            json.dump(default_content, f)
+                        logger.info(f"⚠️ 发现空文件 {file_name}，已重置为默认值")
+                except:
```

Ruff (0.15.2)

[error] Do not use bare `except`

(E722)